### DISCUSSION TITLE
Suggestions on optional features, fso, services and privacy

### DISCUSSION THREAD

**Discussion by ghost** [builtbybel/Bloatynosy#79] [Ideas]
- Windows 11 comes with lots of *optional features*, most of which are bloatware, such as Math Recognizer, Extended Theme Content, Fax and Scan, Internet Explorer Mode, etc. 

- Disable *full screen optimisations* system-wide: Fso reduces game performance and increases input latency for most people. It is recommended to disable fso in almost all game-related Windows guides.

- Add an option to set many services to *manual*, like the ctt utility does.

- Disable *wifi sense*

- Disable mouse acceleration 

Guides that might be helpful:
https://github.com/djdallmann/GamingPCSetup/tree/master/CONTENT/DOCS/NETWORK
https://docs.google.com/document/d/1c2-lUJq74wuYK1WrA_bIvgb89dUN0sj8-hO3vqmrau4/edit

**Comment by e-t-l** [comment]
What exactly does "Extended theme content" do?
